sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj

```diff
@@ -7,22 +7,17 @@
 		<PackageId>Microsoft.Azure.Management.Peering</PackageId>
 		<Description>Microsoft Azure Management Peering library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Peering</AssemblyName>
-		<Version>1.0.1-preview</Version>
-		<PackageTags>Azure Peering;</PackageTags>
+		<Version>2.0.0-preview</Version>
+		<PackageTags>Azure Peering;Peering_2020-01-01-preview;</PackageTags>
     <PackageReleaseNotes>
-	<![CDATA[New features:   Adding Peering Service operations. ]]>
+	<![CDATA[New features:   Adding Registered ASN operations. ]]>
```

Comment:
Please list your change logs in one `<![CDATA[` ... `]]>`  block. You can take Network's csproj as an example